.Lib9c.Tests/Model/State/HammerPointStateTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Lib9c.Tests.Model.State
+{
+    using System.Collections.Generic;
+    using Bencodex.Types;
+    using Libplanet;
+    using Libplanet.Crypto;
+    using Nekoyume.Model.State;
+    using Xunit;
+
+    public class HammerPointStateTest
```

Comment:
Can you please add a test that use `UpdateHammerPoint()` method?

Reply:
OK I'll add that. THX